plugins/catalog-react/src/components/EntityOwnerPicker/useFacetsEntities.ts

```diff
@@ -41,17 +44,37 @@ type FacetsInitialRequest = {
  * hook, which is also used by EntityOwnerPicker.
  * In this mode, the EntityOwnerPicker won't show detailed information of the owners.
  */
-export function useFacetsEntities({ enabled }: { enabled: boolean }) {
+export function useFacetsEntities({
+  filters,
+  enabled,
+}: {
+  filters?: DefaultEntityFilters;
+  enabled: boolean;
+}) {
   const catalogApi = useApi(catalogApiRef);
 
-  const [facetsPromise] = useState(async () => {
+  const facetsPromise = useMemo(async () => {
```

Comment:
this was meant to be created only once. I guess we can remove `facetsPromise` completely and move all the logic inside `useAsyncFn` below.